src/fidesops/ops/models/privacy_request.py

```diff
@@ -806,6 +806,44 @@ class ConsentRequest(Base):
         back_populates="consent_request",
     )
 
+    def cache_identity_verification_code(self, value: str) -> None:
+        """Cache the generated identity verification code for later comparison."""
+        cache: FidesopsRedis = get_cache()
+        cache.set_with_autoexpire(
+            f"IDENTITY_VERIFICATION_CODE__{self.id}",
+            value,
+            config.redis.identity_verification_code_ttl_seconds,
+        )
+
+    def get_cached_identity_data(self) -> Dict[str, Any]:
+        """Retrieves any identity data pertaining to this request from the cache."""
+        prefix = f"id-{self.id}-identity-*"
+        cache: FidesopsRedis = get_cache()
+        keys = cache.keys(prefix)
+        return {key.split("-")[-1]: cache.get(key) for key in keys}
+
+    def get_cached_verification_code(self) -> Optional[str]:
+        """Retrieve the generated identity verification code if it exists"""
+        cache = get_cache()
+        values = cache.get_values([f"IDENTITY_VERIFICATION_CODE__{self.id}"]) or {}
```

Comment:
What's the rationale for not using [`.get(key)`](https://redis-py.readthedocs.io/en/stable/commands.html#redis.commands.cluster.RedisClusterCommands.get) here?

Reply:
This is all just a direct copy from `PrivacyRequest` above